_data/simple-icons.json

```
@@ -6572,6 +6572,11 @@
             "source": "https://www.mixcloud.com/about",
             "guidelines": "https://www.mixcloud.com/about"
         },
+        {
+            "title": "Mixpanel",
+            "hex": "111111",
```

I think the hex value should be `#7856FF` as that's the primary color according to the [press kit](https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9).

```suggestion
            "hex": "7856FF",
```